.circleci/config.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - run:
           name: Type Checking
           command: yarn run type-check
+      - run:
+          name: Type Export
+          command: yarn run build:types
```

Comment:
Does it make sense to add this to the linting step? I think I would have expected this in the build step

Reply:
build is to test the bundle size though, maybe it should be a fully separate step for typescript to both test the build and export?

Reply:
We could run `build` and `build:types` in the job, and run `test:size` in another job after the `build` job? We do something similar in the Autocomplete CI: https://github.com/algolia/autocomplete/blob/822775c5754ebbd7036658b550e77b9cc966acdb/.circleci/config.yml#L202-L204

Reply:
Changed it to build, it makes more sense that way indeed